src/twisted/newsfragments/12096.feature

```diff
@@ -0,0 +1 @@
+Include client port in WSGI response
```

Comment:
Is this a request or a response ?
```suggestion
twisted.web.wsgi request environment now containes the remote peer port number.
```

Reply:
typo in "containes" and "remote peer" is redundant. :wink:

Reply:
updated :) thanks for the check.

Reply:
You are correct, thanks!